services/api/src/emails/layout.html

```diff
@@ -281,8 +286,8 @@
           <table class="email-content" width="100%" cellpadding="0" cellspacing="0" role="presentation">
             <tr>
               <td class="email-masthead">
-                <a href="{{&appUrl}}" class="f-fallback email-masthead_name">
-                  <img src="{{&appLogoUrl}}" alt="{{appName}}" />
+                <a href="{{&APP_URL}}" class="f-fallback email-masthead_name">
+                  <img src="{{&APP_LOGO_URL}}" alt="{{APP_NAME}}" class="email-masthead_logo" />
```

Comment:
is the `&` still needed ?

Reply:
perhaps not you can do either way... I'm fine to remove if you prefer